contracts/interfaces/IPriceOracle.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity 0.7.6;
+
+
+interface IPriceOracle {
+  function conversionRate(
```

Comment:
Pls specify that if token is not listed, this should be revert or return 0.

Reply:
added